.dockerignore

```diff
@@ -1,12 +0,0 @@
-.git
-**/_cache
-app
-build-image
-cni
-contrib
-daemonset
-dashboard
-Documentation
-hack
-utils
-vendor
```

Comment:
Is there a reason to get rid of this file? My thought is that it would ensure that certain directories would still not be picked up in the event that a container is ever built manually.

Does using the multi-stage build below offer the same protections from stuff accidentally slipping into the container image?

Reply:
As we're using now multi-stage it shouldn't be needed anymore.